backend/backend.py

```diff
@@ -181,7 +181,7 @@ def send_recording_status(self) -> None:
     def send_recording_error(self) -> None:
         """ Send the most recent exception to the recording status MQTT topic """
```

Comment:
Since you are making small changes, could you put a full stop at the end of this sentence? I must've missed it in my last PR.

Reply:
Done
24d52ef